### DISCUSSION TITLE
Username 666 easter egg not an issue just an idea

### DISCUSSION THREAD

**Discussion by akivec** [ftde0/yt2009#182] [Ideas]
You can add username 666 into yt2009 when you type in /666 it says the account is suspended but after refreshing a few times it becaumes like in the vid this would be a cool easter egg

**Comment by ftde0** [comment]
would be kinda funny but not sure whether something like that is something actually worth development time and would be actually used

**Comment by minihoot** [comment]
> would be kinda funny but not sure whether something like that is something actually worth development time and would be actually used

what about making /666 just go to the video?

**Comment by Pwnz0rserver2009** [comment]
if this was added, wouldn't it be inaccurate to actual old youtube?

**Comment by minihoot** [comment]
sure but like if you’re going to /666 i doubt you’d care and it’s not like
it matters
